mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1649,6 +1649,109 @@ export const getActions = (
             searchTerms: ['docs', 'documentation', 'help', 'support'],
             tooltip: "Documentation, tutorials, and how-tos on all functionality in Mito."
         },
+        [ActionEnum.Select_Columns]: {
+            type: 'build-time',
+            staticType: ActionEnum.Select_Columns,
+            longTitle: 'Select column',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                // We close the editing taskpane if its open
+                closeOpenEditingPopups();
+
+                // We open the column selection taskpane
+                let newStartingColumnIndex = gridState.selections[0].startingColumnIndex;
+                let newEndingColumnIndex = gridState.selections[0].endingColumnIndex;
+                if (newStartingColumnIndex === -1) {
+                    newStartingColumnIndex = 0;
+                }
+                if (newEndingColumnIndex === -1) {
+                    newEndingColumnIndex = sheetData.numColumns - 1;
+                }
```

Comment:
Yeah, so there's currently a bug in this logic -- it doesn't appear to be broken in rows, but I think this check is wrong with columns. See attached video. 

https://github.com/mito-ds/mito/assets/21002431/f9dc5786-926a-4b85-93fb-df6a23b36a33